pkg/vault/contracts/Authorizer.sol

```diff
@@ -129,6 +132,10 @@ contract Authorizer is IAuthorizer {
      */
     function setDelay(bytes32 action, uint256 delay) external {
         _require(msg.sender == address(this), Errors.SENDER_NOT_ALLOWED);
+
+        bytes32 setAuthorizerId = vault.getActionId(IVault.setAuthorizer.selector);
```

Comment:
This is a fixed value so why not store the action id instead of the vault's address in order to query it each time?

Reply:
I thought about that too... the problem is that the interface does not ensure it is actually constant, it would be wrong conceptually speaking.